source/support.html.erb

```diff
@@ -92,7 +95,84 @@ title: Support
         the next working day.
       </p>
 
-      <h2 class="heading-large">Contacting us about a critical issue</h2>
+      <h2 class="heading-large">SLAs for production tenants</h2>
+
+      <table>
+        <tbody>
+          <tr>
+          <th>Classification</th>
+          <th>Example</th>
+          <th>In hours</th>
+          <th>Out of hours</th>
+        </tr>
+        <tr>
+          <th scope="row">P1<br/>Critical<br/>Incident</th>
+          <td>
+             <ul class="list list-bullet">
+               <li  class="font-xsmall">Apps no longer being served due to an issue with our platform</li>
+               <li class="font-xsmall">Serious security breach on the platform</li>
+               <li class="font-xsmall">You are unable to push an emergency fix to an app due to the PaaS API not being available</li>
+               <li class="font-xsmall">Your live production app has a P1 issue which cannot be resolved without us</li>
```

Comment:
Your live production app has a critical P1 issue which cannot be resolved without help from us